modules/tracker/models/base/ScalarModelBase.php

```diff
@@ -32,6 +32,7 @@ public function __construct()
             'scalar_id' => array('type' => MIDAS_DATA),
             'trend_id' => array('type' => MIDAS_DATA),
             'user_id' => array('type' => MIDAS_DATA),
```

Comment:
We should have a more in-depth discussion down the road. I think that moving user_id out to submission should be part of a larger model change.